src/index.html

```diff
@@ -11,6 +11,41 @@
     <link rel="stylesheet" href="./style.css">
   </head>
   <body>
-    <h1>Search bar airbnb</h1>
+    <div class="container">
+      <form
+        class="formBig"
+        method="get"
+        action="/"
+        data-qa="big"
+      >
+        <label for="keypress">
```

Comment:
The `for` attribute is redundant here if you wrap the input inside the label. So, you can remove it. Just add class names for styling 
```suggestion
        <label>
```